content/blog/0012-effect-3.0.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: Effect 3.0 (Release)
+excerpt: Production-Grade Software in TypeScript with Effect
+date: 2024-03-16
+authors:
+  - name: Michael Arnaldi
+    description: Chief Executive Officer
+    avatar: https://pbs.twimg.com/profile_images/1232338149340319751/Ln9nNWyk_400x400.jpg
+    twitter: https://twitter.com/MichaelArnaldi
+---
+
+## Announcement
+
+Effect is finally API Stable!
+
+I am pleased to announce that after 5 years of work and 3+ years of production usage we are ready to release Effect 3.0, the first stable release of the core of the Effect Ecosystem.
+
+Starting with 3.0 the main package `effect` will follow semantic versioning:
+
+1) major releases will include breaking changes
+2) minor releases will include new features and new modules
+3) patch releases will include bug-fixes
+
+We do not expect new major releases in the near future, we will only release a new major when a significant number of API improvements have been made and after a long period of feedback (6-12 months).
```

Comment:
A major/year is a scary pace for folk who quiver at any accidental version updates and conceptualize a 'major' as "everything is now totally different, updating means upgrading everything, new syntax new api new gotchas. Living on the edge baybeeeeee". I know it's clarified before and here again, but the differentiation between an iteration and a whole new release should be touched upon

also, the 6-12 period is overloaded in this post, it has
- long period of feedbacks
- development of 'higher and higher levels of abstraction'
- effect/cluster

note it's a lot to promise

Reply:
they relate to different projects, in 6-12 months we expect:
- effect cluster to be usable (currently in alpha)
- platform & co to be stable (they are already in use in prod) 
- we might release a new effect major with incorporated feedback

Note that pretty much every major JS project has a major every year, a major doesn't mean break the world